tests/rngtest/main.c

```diff
@@ -0,0 +1,347 @@
+/*
+ * Copyright (C) 2015 Jan Wagner <mail@jwagner.eu> 
+ * Copyright (C) 2015 Freie Universität Berlin
```

Comment:
Quick question: do you work for Freie Universität Berlin?
If not then FU-Berlin does not hold a copyright claim on this...